src/app/services/business.service.ts

```diff
@@ -66,45 +65,44 @@ export class BusinessService implements OnInit {
 
```

Comment:
Please remove this inmemory service from this file. We already have inmemory business service in trappsdb service. Please make changes in that.